frontend/src/pages/distributedWorkloads/global/projectMetrics/sections/RequestedResources.tsx

```diff
@@ -58,7 +58,7 @@ const RequestedResourcesBulletChart: React.FC<RequestedResourcesBulletChartProps
   });
 
   const requestedByThisProjectData = getDataItem({
-    name: `Requested by ${namespace}`,
+    name: `Requested by ${projectDisplayName}`,
```

Comment:
In your current state this is optional -- this is something you should have checked for. We need to make sure we trust our types 100% and there is no ambiguity. I wonder if we can get a linter for this 🤔 no `undefined` in string construction.

Logged this https://issues.redhat.com/browse/RHOAIENG-6361 -- you're going to make yours required, you don't need to worry about this per se.